cypress/support/index.js

```diff
@@ -14,7 +14,16 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import './commands'
+import "./commands";
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+const sizes = [
+  "iphone-x",
+  ["iphone-x", "landscape"],
+  ["ipad-2", "landscape"],
```

Comment:
ain't ipad 2 landscape pretty much the same as macbook? maybee better with just ipad 2 no landscape?

Reply:
Yeah ok, kinda close, sure I can update it.

Reply:
Fixed, had to remove some switch statements for the portrait version